vulnerability/events_creator.go

```diff
@@ -391,6 +391,20 @@ func (e EventsCreator) getCVSSScore(vul trivyTypes.DetectedVulnerability) float6
 	return getCVSSValue(vul, func(cvss dbTypes.CVSS) float64 { return cvss.V2Score }, zeroVal)
 }
 
+const NVDVulnDetailBaseURL = "https://nvd.nist.gov/vuln/detail/"
+
+func getReference(vul trivyTypes.DetectedVulnerability) string {
+	if vul.CVSS == nil {
+		return vul.PrimaryURL
+	}
+
+	if _, ok := vul.CVSS[trivyVul.NVD]; ok {
```

Comment:
I'm assuming that if there is a NVD in CVSS then this vulnerability exists in NVD database and the link will be valid. It's a bit brittle in my opinion, but if we want to replace aquasec links that's the simplest way I could find

Reply:
Also, as far as I understand this is the same solution we have today in kibana, right?

Reply:
yes, exactly!

Reply:
nit: invert the `if`
```
if _, ok := vul.CVSS[trivyVul.NVD]; !ok {
  return vul.PrimaryURL
}
```